examples/match-statement-with-maps/match_statement_with_maps.md

```diff
@@ -1,10 +1,6 @@
 # Match statement with maps
 
-Match statement can be used to match maps.
-Patterns on the left hand side in a match statement can have variable
-parts that can be captured.
-Useful for working directly with `json`.
-Match semantics are open (may have fields other than those specified in the pattern).
+Match statement can be used to match maps. Patterns on the left-hand side in a match statement can have variable parts that can be captured. Useful for working directly with `json`. Match semantics are open (may have fields other than those specified in the pattern).
```

Comment:
```suggestion
A `match` statement can be used to match maps. Patterns on the left-hand side in a match statement can have variable parts that can be captured. It is useful for working directly with `json`. Match semantics are open (may have fields other than those specified in the pattern).
```

Reply:
Fixed in https://github.com/ballerina-platform/ballerina-distribution/pull/2930/commits/850aaadbbb3db21491d1ca85c58cf87a6dc589e8.